src/applications/pensions/config/form.js

```diff
@@ -1056,7 +1060,12 @@ const formConfig = {
                   properties: {
                     childInHousehold:
                       dependents.items.properties.childInHousehold,
-                    childAddress: dependents.items.properties.childAddress,
+                    childAddress: merge(
```

Comment:
can just do `childAddress: addressSchema({ ...` here. should be the same in this case as its returning a new object.